aggregator/aggregator.py

```diff
@@ -10,86 +10,107 @@
 # Project imports
 import log_manager
 
-# Constants
-SECONDS_PER_DAY = 86400
-SERVER_LIST_FILE = '../servers.txt'
+class Aggregator:
+  # Constants
+  SECONDS_PER_DAY = 86400
+  SERVER_LIST_FILE = '../servers.txt'
```

Comment:
this class variable is shared by all instances. you can access it from an instance or with Aggregator.SECONDS_PER_DAY